### ISSUE TITLE
ClipId not being returned in the result of UploadEntireFileAsync

### ISSUE THREAD

**Issue by Robson-Rocha** [mfilippov/vimeo-dot-net#190]
Until recently, everything was working fine. 

Suddenly, all uploads are not returning the generated ClipId. This behavior was first observed in v3.10 but was also observed in v3.2.0.

My call:

```
using var videoFile = new BinaryContent(recordingPath);
var uploadRequest = await VimeoClient.UploadEntireFileAsync(videoFile, (long)1e+7, statusCallback: percent => {
    Console.WriteLine($"{percent}% uploaded");
});
Console.WriteLine(uploadRequest.ClipId); // returns null
```

I need the ClipId for further requests to MoveVideoToFolder, UpdateVideoMetadataAsync, AllowEmbedVideoOnDomainAsync and finally GetVideoAsync (to get the player's embedded URL).

**Comment by Robson-Rocha** [comment]
I found and fixed the issue in PR #191

**Comment by gtsoftwareltd** [comment]
I'm also using UploadEntireFileAsync and found that the result's ClipId is null

In the debugger, when the result 'IsVerifiedComplete' then result.Ticket.Id.Value gives the ClipId for use in other calls.

The pull request might be a better option, but this is working for me with the released v3.2.0

**Comment by Robson-Rocha** [comment]
@gtsoftwareltd This is a good workaround, good find! But the PR is a better option because it doesn't break existing functionality that relies on the ClipId property.

**Comment by mfilippov** [comment]
Fixed in 3.2.2
